current-sprint.js

```diff
@@ -1,22 +1,32 @@
 exports.getCurrentSprint = function(ghdata) {
-  // We want the current milestone, plus its start and end date.
-  // We find this milestone by looking for the oldest one in the list :)
+  // For specific milestone queries, we get this from the initial milestoneDueOn
 
-  let phase = ghdata.repository.milestones.edges.reduce ((a, m) => {
-    if(m.node.dueOn == null) return a;
-    if(a == null || a.node.dueOn == null) return m;
-    if(new Date(a.node.dueOn) < new Date(m.node.dueOn)) return a;
-    return m;
-  });
-  if(phase == null) {
-    return null;
+  let phase = null;
+
+  if(ghdata.milestoneDueOn) {
```

Comment:
When I ran node code.js, I got this
```
 UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'milestoneDueOn' of undefined
    at Object.exports.getCurrentSprint (D:\src\sites\status.keyman.com\current-sprint.js:6:13)
```

Reply:
ah, likely cause I haven't set up tokens on this machine

Reply:
Yeah, some of the error case pathways are not complete for this site; don't really want to expend too much effort on them for now.